app/controllers/mfa_confirmation_controller.rb

```diff
@@ -45,4 +49,12 @@ def handle_max_password_attempts_reached
     sign_out
     redirect_to root_url, flash: { error: t('errors.max_password_attempts_reached') }
   end
+
+  def prevent_page_render
+    if IdentityConfig.store.select_multiple_mfa_options
+      auth_method_confirmation_url
+    else
+      render :file => "views/mfa_confirmation/new.html.erb", :layout => true, :status => 302
```

Comment:
How come this is a 302 status (usually a redirect?) but it's rendering instead of redirecting?

Also can we use new-style hashes `render file: "abc/def", layout: true` if possible?

Reply:
yeah, I thought redirecting would cause it not render, but I was wrong. Instead, we are going to put a flag on that route.